GTViewController/Classes/Nav Classes/GTViewController.m

```diff
@@ -731,7 +730,7 @@ - (void)configureToolbar {
     [buttons addObject:[[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFlexibleSpace target:nil action:nil]];
     
     [self.navToolbar setItems:[NSArray arrayWithArray:buttons] animated:YES];
-    
+    self.navToolbar.tintColor = [UIColor whiteColor];
```

Comment:
This line needs to have a if respondsToSelector statement around it because we support iOS6 still. @ryancarlson could you do that when you release the new version of the pod?